### ISSUE TITLE
How to get the extrinsics/intrinsics/conf?

### ISSUE THREAD

**Issue by douyh** [ByteDance-Seed/Depth-Anything-3#38]
I use the example code from huggingface

```
import torch
from depth_anything_3.api import DepthAnything3

# Load model from Hugging Face Hub
device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
model = DepthAnything3.from_pretrained("depth-anything/da3metric-large")
model = model.to(device=device)

# Run inference on images
images = ["image1.jpg", "image2.jpg"]  # List of image paths, PIL Images, or numpy arrays
prediction = model.inference(
    images,
    export_dir="output",
    export_format="glb"  # Options: glb, npz, ply, mini_npz, gs_ply, gs_video
)

# Access results
print(prediction.depth.shape)        # Depth maps: [N, H, W] float32
print(prediction.conf.shape)         # Confidence maps: [N, H, W] float32
print(prediction.extrinsics.shape)   # Camera poses (w2c): [N, 3, 4] float32
print(prediction.intrinsics.shape)   # Camera intrinsics: [N, 3, 3] float32

```

Something wrong with the export_formt="glb".

I can't get extrinsics/intrinsics.

After I remove this line:

```
    export_format="glb"  # Options: glb, npz, ply, mini_npz, gs_ply, gs_video
```

prediciton.conf, prediction.extrinsics, prediction.intrinsics are `None`

How can I get the correct conf, extrinsics, intrinsics?

**Comment by sourcesync** [comment]
Hi @douyh did you get any resolution on this?  I'm also getting None.
